docs/components/badge.md

```diff
@@ -22,12 +22,14 @@ The Badge component can take the following appearance values:
 - `neutral`
 - `custom`
 
-<KBadge appearance="success" class="mr-2">SUCCESS</KBadge>
-<KBadge appearance="warning" class="mr-2">WARNING</KBadge>
-<KBadge appearance="danger" class="mr-2">DANGER</KBadge>
-<KBadge appearance="info" class="mr-2">INFO</KBadge>
-<KBadge appearance="neutral" class="mr-2">NEUTRAL</KBadge>
-<KBadge>DEFAULT</KBadge>
+<div class="vertical-spacing">
```

Comment:
nit: this class name doesn't make sense -- shouldn't it be horizontal?

Reply:
Fixed